### ISSUE TITLE
ComponentClass<P>' requires between 0 and 1 type arguments

### ISSUE THREAD

**Issue by Sandip2110** [bkrem/react-d3-tree#188]
getting this error as in the react-d3-tree.d.ts file for
var Tree: React.ComponentClass<ReactD3TreeProps,any>;

There should be 1 argument

**Comment by bkrem** [comment]
Hi @Sandip2110,

I couldn't reproduce this on my side since technically `var Tree: React.ComponentClass<ReactD3TreeProps, any>` is valid (but redundant), since 

```ts
// @types/react/index.d.ts
type ComponentState = any;
interface ComponentClass<P = {}, S = ComponentState>
```

I removed the `any` declaration so you shouldn't be seeing this anymore, although I'd still be curious how and when you saw this error if you have a moment :)

**Comment by bkrem** [comment]
Published as [v1.12.3](https://github.com/bkrem/react-d3-tree/releases/tag/v1.12.3)
